docs/reference/ocm_add_sources.md

```diff
@@ -483,7 +483,7 @@ are handled:
     The size of the blob
   
   
-  Options used to configure fields: <code>--mediaType</code>, <code>--size</code>, <code>--digest</code>, <code>--reference</code>
+  Options used to configure fields: <code>--size</code>, <code>--digest</code>, <code>--reference</code>, <code>--mediaType</code>
```

Comment:
Can we fix the generated stuff to be consistent, or run only when we have relevant changes? (just open a new issue about it)

Reply:
That's a good point, it is sign for an error in the old code. It seems to iterate over a map, which doesn't not provide a stable order, so every generation causes another order. I'll fix this.